docs/Formatting.md

```diff
@@ -96,6 +87,18 @@ Here `foo` is categorized as `mixed content`, since it contains text and tag ele
 
 ***
 
+### xml.format.enabled
+
+  Set to `false` to disable XML formatting. Defaults to `true`.
+
+***
+
+### xml.format.legacy
+
+  Set to `true` to enable legacy formatter. Defaults to `false`.
+
+  Any setting unsupported by the legacy formatter will be marked with **Not supported by the legacy formatter**, while settings exclusive to the legacy formatter will be marked with **This setting is only available with legacy formatter**.
```

Comment:
... will be marked with **Not supported by the legacy formatter** *in this document and in the settings*, ...